Kitodo-Validation/pom.xml

```diff
@@ -0,0 +1,81 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+ *
+ * (c) Kitodo. Key to digital objects e. V. <contact@kitodo.org>
+ *
+ * This file is part of the Kitodo project.
+ *
+ * It is licensed under GNU General Public License version 3 or later.
+ *
+ * For the full copyright and license information, please read the
+ * GPL3-License.txt file that was distributed with this source code.
+ *
+-->
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <groupId>org.kitodo.validation</groupId>
+    <artifactId>kitodo-validation</artifactId>
+    <parent>
+        <artifactId>kitodo-production</artifactId>
+        <groupId>org.kitodo</groupId>
+        <version>3.0-SNAPSHOT</version>
+    </parent>
+    <organization>
+        <name>kitodo.org</name>
+        <url>http://kitodo.org</url>
+    </organization>
+    <name>Kitodo - Validation</name>
+    <url>https://github.com/kitodo/kitodo-production</url>
+
+    <properties>
+        <parent.basedir>${basedir}/../</parent.basedir>
+        <checkstyle.config.location>${parent.basedir}/config/checkstyle.xml</checkstyle.config.location>
+    </properties>
+
+    <build>
```

Comment:
- Are all build statements needed? There are no tests nor resource directories nor xml files....
- On the other side the created module is not copied to module directory (Kitodo/modules or /usr/local/kitodo/modules as defined in the kitodo_config.properties file)